stories/dynamic-page/dynamic-page.stories.js

```diff
@@ -0,0 +1,929 @@
+                    <a class="fd-tabs__link" aria-controls="fuCwV554" href="#fuCwV554" role="tab">
+                        <span class="fd-tabs__tag">
+                            Link
+                        </span>
+                    </a>
+                </li>
+                <li role="listitem" class="fd-tabs__item">
+                    <a class="fd-tabs__link" aria-controls="AiWfz169" aria-selected="true" href="#AiWfz169" role="tab">
+                        <span class="fd-tabs__tag">
+                            Selected
+                        </span>
+                    </a>
+                </li>
+            </ul>  
+        </div>     
+        <div class="fd-dynamic-page__content fd-dynamic-page__content--s">
+            <div class="fd-tabs__panel" aria-expanded="false" id="fuCwV554" role="tabpanel">
+                Lorem ipsum dolor, sit amet consectetur adipisicing elit. Rerum illo voluptatem, unde voluptate aliquam 
+                eum tempore aliquid dolorem cumque ullam perspiciatis omnis et asperiores dolores, consequatur, suscipit sint animi amet?
+                Lorem ipsum dolor, sit amet consectetur adipisicing elit. Rerum illo voluptatem, unde voluptate aliquam eum tempore aliquid 
+                dolorem cumque ullam perspiciatis omnis et asperiores dolores, consequatur, suscipit sint animi amet?
+            </div>
+            <div class="fd-tabs__panel" aria-expanded="true" id="AiWfz169" role="tabpanel">
+                <h4>This is an example of a collapsed title area and header and instead of displaying a Summary Line</h4>
+                <br />
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+            </div>
+        </div>
+        <div class="fd-bar fd-bar--floating-footer">
+            <div class="fd-bar__right">
+                <div class="fd-bar__element">
+                    <button aria-label="button" class="fd-button fd-button--emphasized fd-button--compact">Save</button>
+                </div>
+                <div class="fd-bar__element">
+                    <button aria-label="button" class="fd-button fd-button--transparent fd-button--compact">Cancel</button>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<br /><br />
+`;
+
+mobile.storyName = 'Mobile mode';
+mobile.parameters = {
+    docs: {
+        iframeHeight: 150,
+        storyDescription: 'The dynamic page with Mobile specification(S). Use the `--s` modifier on `title-area`, `breadcrumb` ' +
```

Comment:
The dynamic page displays on mobile screens with the S size. To optimize dynamic page for mobile, add these modifier classes to the following elements:

Element | Modifier class
:-- | :--
Title Area | `fd-dynamic-page__title-area--s`
Header | `fd-dynamic-page__header--s`
Content | `fd-dynamic-page__content--s`
Summary line (optional) | `fd-dynamic-page__summarized-title-area--s`

It is recommended to add a summary line insead of the expand/collapse header feature so users can better focus on the page content. To display a summary line, add `fd-dynamic-page__summarized-title` in the `fd-dynamic-page__summarized-title-area` element.

Reply:
Modified, and added a row for breadcrumb modifier here as for mobile, there is a specific padding applied for the breadcrumb element.